presto-docs/src/main/sphinx/sql/show-create-table.rst

```diff
@@ -0,0 +1,33 @@
+=================
+SHOW CREATE TABLE
+=================
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW CREATE TABLE table_name
+
+Description
+-----------
+
+Display the sql statement that created the table.
+
+Examples
+--------
+Show create table statement for ``orders``::
+
+    presto> SHOW CREATE TABLE sf1.orders;
+                  Create Table
```

Comment:
Nit: the header width doesn't seem to match the width of the SQL. Please re-run and copy the exact output.